Doc/library/functions.rst

```diff
@@ -1181,12 +1181,12 @@ are always available.  They are listed here in alphabetical order.
    *buffering* is an optional integer used to set the buffering policy.  Pass 0
    to switch buffering off (only allowed in binary mode), 1 to select line
    buffering (only usable in text mode), and an integer > 1 to indicate the size
```

Comment:
As you've added to the list, remove the previous joining 'and'. I also might consider reformatting this list as a definition list or a bulleted list as there are now four items (-1, 0, 1, >=2)
```suggestion
   buffering (only usable in text mode), an integer > 1 to indicate the size
```